signbank/dictionary/models.py

```diff
@@ -302,7 +302,7 @@ def __str__(self):
         return self.english_name
 
     class Meta:
-        ordering = ['field', 'machine_value']
+        ordering = ['field', 'english_name', 'machine_value']
```

Comment:
```suggestion
        ordering = ['field', 'english_name']
```

Given that our machine values are randomly generated, I think we should be clear here that there is no utility in using the machine value for ordering.

Reply:
Absolutely. Commit done.